src/palimpzest/datamanager.py

```diff
@@ -26,7 +26,7 @@ class DataDirectorySingletonMeta(type):
     def __call__(cls, *args, **kwargs):
         with cls._lock:
             if cls not in cls._instances:
-                instance = super(DataDirectorySingletonMeta, cls).__call__(*args, **kwargs)
+                instance = super().__call__(*args, **kwargs)
```

Comment:
Confident these are equivalent (the singleton metadata pattern is a popular Pythonic thing to do since it does not support singletons as cleanly as other languages with nice inversion of control mechanisms),

the below is a standard guide for setting up a singleton in python

```Python
from threading import Lock

class Singleton(type):
    _instances = {}
    _lock = Lock()
    
    def __call__(cls, *args, **kwargs):
        with cls._lock:
            if cls not in cls._instances:
                cls._instances[cls] = super().__call__(*args, **kwargs)
        return cls._instances[cls]

# Usage example:
class MyClass(metaclass=Singleton):
    def __init__(self):
        self.value = 0
```

or, equivalently

```python
def singleton(cls):
    _instances = {}
    _lock = Lock()
    
    def get_instance(*args, **kwargs):
        with _lock:
            if cls not in _instances:
                _instances[cls] = cls(*args, **kwargs)
        return _instances[cls]
        
    return get_instance

# Usage example:
@singleton
class MyClass:
    def __init__(self):
        self.value = 0
```